### DISCUSSION TITLE
Dynamic routing using KV store

### DISCUSSION THREAD

**Discussion by AlexMikhalev** [poem-web/poem#481] [Q&A]
Is it possible to create routes and subdomains using user.maindomain.ltd for each user? I would like to create a separate subdomain for each user. Similar to https://github.com/fabiolb/fabio, which is using Consul KV for routing.

**Comment by sunli829** [comment]
Use this

https://docs.rs/poem/1.3.52/poem/struct.RouteDomain.html

**Comment by AlexMikhalev** [comment]
Thank you Sunli, that superb; instead of moving to go based stack I can stick to Rust-based infrastructure.
